### PR TITLE
update to Adminer 4.7.0

### PR DESCRIPTION
**Pull request by mnsuccess**
Update to Adminer 4.7.0

### CODE REVIEW

File: sources.download [hunk 1]
```diff
@@ -1 +1 @@
-ca7c418ebf8e45eb1f95a03c26cf0e0db77bd256b425087afe329ab6455f3a72  adminer-4.6.3.tar.gz https://github.com/vrana/adminer/releases/download/v4.6.3/adminer-4.6.3.tar.gz
+ca7c418ebf8e45eb1f95a03c26cf0e0db77bd256b425087afe329ab6455f3a72  adminer-4.7.0.tar.gz https://github.com/vrana/adminer/releases/download/v4.7.0/adminer-4.7.0.tar.gz
```
Comment (fabiomontefuscolo) [inline sources.download]:
```suggestion
d335105d1b45d373eb8f8edf0cbfb466  adminer-4.7.0.tar.gz https://github.com/vrana/adminer/releases/download/v4.7.0/adminer-4.7.0.tar.gz
```

File: adminer-elasticsearch.spec [hunk 2]
```diff
@@ -50,6 +50,9 @@ rm -rf $RPM_BUILD_ROOT
 /usr/clearos/sandbox/etc/httpd/conf.d/adminer-elasticsearch.conf
 
 %changelog
+* Mon Jan 07 2019 eGloo <developer@egloo.ca> 4.7.0
```
Comment (marclaporte) [inline adminer-elasticsearch.spec]: @Manasoft404: you are updating. Why are you putting eGloo?

Comment (mnsuccess) [reply adminer-elasticsearch.spec]: Should I change and put my email address?

Comment (marclaporte) [reply adminer-elasticsearch.spec]: yes

**Comment by mnsuccess** [comment]
I'm still waiting for the validation of this pull request

**Comment by pcbaldwin** [comment]
Looks good.  Keep in mind, the build workflow hooks into the ClearCenter build system.  I no longer have access to that system, so I'll try to bring this issue to their attention.

**Pull request merged**